### ISSUE TITLE
Incorrect stats ?

### ISSUE THREAD

**Issue by andricop** [mhdhejazi/CoronaTracker#115]
![095EDB98-BB0D-4DA5-9487-ACBC5CD76332](https://user-images.githubusercontent.com/64187262/82126435-1d019200-97ad-11ea-8486-357a3e43a91c.jpeg)

Why is France not appearing here ? It’s more affected than Brazil or Russia.

Thanks

**Comment by mhdhejazi** [comment]
You are right. What you're seeing here is still the most affected countries (in terms of total cases) but showing how many deaths instead. I guess we have to improve this chart and change the order.
